### DISCUSSION TITLE
Allow text input for TimeStamp fields in Keystone UI

### DISCUSSION THREAD

**Discussion by BradNut** [keystonejs/keystone#5307] [Feature requests]
At the moment tagging a schema with type of `timestamp()` creates a UI field which only allows entry via a calendar. 
If you need to input a calendar date that is far in the past, let's say a person's birthday, you need to click all the way back on the calendar to get to that date.

This is time consuming if there are a lot of entries that need to be created whose timestamp is greater than a few months.

My current dependencies I am using are the following, so if newer packages contain this feature let me know:
```
    "@keystone-next/admin-ui": "^8.0.1",
    "@keystone-next/auth": "^14.0.0",
    "@keystone-next/cloudinary": "^2.0.9",
    "@keystone-next/fields": "^4.1.1",
    "@keystone-next/keystone": "^9.3.0",
    "@keystone-next/types": "^12.0.0",
```

**Comment by bladey** [comment]
Hey @BradNut, you're right, we only have `timestamp` right now, we'll be improving this going forward, thanks for calling it out.
